index.js

```diff
@@ -110,16 +107,13 @@ exports.main = function main() {
       .then(() => getProjectReadme(projectDirName))
       .then(projectReadme => fs.writeFile(path.join(projectPath, "README.md"),
                                           stripAnsi(projectReadme)))
-      .then(() => getProjectCreatedMessage(projectPath))
-      .then(console.log);
+      .then(() => getProjectCreatedMessage(projectPath));
```

Comment:
@saintsebastian we should probably return a more useful object to the caller, e.g. something like:
```js
.then(() => {
  const projectCreatedMessage = getProjectCreatedMessage(projectPath);

  return {projectPath, projectCreatedMessage};
});
```

This should also make much more clear what line 16 of the ` bin/create-webextension` is actually doing.